pkg/apis/v1alpha5/provisioner.go

```diff
@@ -32,63 +34,76 @@ import (
 type ProvisionerSpec struct {
```

Comment:
Synced offline. I'm not sure that `set` annotations are needed throughout this. We should double-check that this actually produces different hashes